packages/flutter/lib/src/foundation/assertions.dart

```diff
@@ -12,6 +16,43 @@ typedef FlutterExceptionHandler = void Function(FlutterErrorDetails details);
 /// and other callbacks that collect information into a string buffer.
 typedef InformationCollector = void Function(StringBuffer information);
 
+/// Property with an [StackTrace] [value] that can be filtered to only show
+/// relevant frames using a XXX.
```

Comment:
Is XXX cruft?

Reply:
Made private and removed.
Note all XXX comments in this CL are places where there is more work I know I need to do.